web/.mocharc.json

```diff
@@ -1,4 +0,0 @@
-{
-  "$schema": "https://json.schemastore.org/mocharc.json",
-  "require": "tsx"
```

Comment:
This no longer seems to be necessary for mocha tests to run, and simplifies the config